src/main/java/restaurant_screens/IFrame.java

```diff
@@ -0,0 +1,7 @@
+package restaurant_screens;
+
+import entities.Restaurant;
+
+public interface IFrame {
+    public void refresh(Restaurant restaurant);
```

Comment:
Refresh will be a general method which everyone can use, so maybe use an object parameter which everyone can cast later to what they need.

Reply:
Agreed this will be changed in later implementations, depending on how general or specific we would like to be.